src/AWS/Signers/V4.elm

```diff
@@ -1,4 +1,4 @@
-module AWS.Signers.V4 exposing (sign)
+module AWS.Signers.V4 exposing (sign, authorization)
```

Comment:
That's ok to expose it for testing. This module is no longer exposed by the package, so none of this will be visible to the client anyhow